CHANGELOG.MD

```diff
@@ -0,0 +1,43 @@
+# Change Log
+## 1.1 Release
+	- Supports Houdini 18.0 and USD 19.11
+	- Uses RPR SDK 1.34.5
+	- Uses RIF 1.4.3
+### New features
+	- Added Centos 7 build
+	- Added RPR Render Settings node for use in Houdini in addition to global per project settings.
+	- Image Gamma supported for GL textures
+	- Add support for alpha compositing render output.  This allows overlaying on the Houdini background and viewport gizmos
+	- Support for UsdTransform2D nodes in shading (Only single level, not nested)
+	- Per object subdivision.  Also added a HDA RPR Mesh Properties Node for specifying subdivision level in Houdini 
+	- Support of doublesided meshes
+	- Support of HdDisplayStyle::displacementEnabled attributes
+	- Camera FStop, Focal Length, Aperature, Focus Distance and Clipping Range is supported on cameras
+	- Support for tapered width curves
+	- Add debugging environment variable.  Set HD_RPR_ERROR_OUTPUT_FILE=some_file.txt to dump errors when rendering with hdRPR.
```

Comment:
I would specify that it works only for USD errors and warnings - all errors that come from TF_RUNTIME_ERROR, TF_CODING_ERROR, TF_WARN, etc. Why? Because they are typically hidden by Houdini and not always printed into "Houdini Console"